src/macros.jl

```diff
@@ -651,7 +651,7 @@ function parseSDconstraint(_error::Function, sense::Symbol, lhs, rhs)
         _error("Invalid sense $sense in SDP constraint")
     end
     vectorized = false
-    parsecode, buildcall = parse_one_operator_constraint(_error, false, Val(:in), aff, :(PSDCone()))
+    parsecode, buildcall = parse_one_operator_constraint(_error, false, Val(:in), aff, :(JuMP.PSDCone()))
```

Comment:
Doesn't this indicate an escaping issue?

Reply:
From the viewpoint of `parse_one_perator_constraint`, the last argument is the set given by the user so it should be in the user scope, not JuMP scope.